packages/memcache-client/src/types.ts

```diff
@@ -49,6 +49,7 @@ export type MemcacheClientOptions = {
   compressor?: CompressorLibrary;
   logger?: any;
   Promise?: PromiseConstructor;
+  tls?: any;
```

Comment:
hi there, what about defining some types for this tls options? nothing fancy

Reply:
Yes, I will use `tls.ConnectionOptions` here.